### ISSUE TITLE
ApiRequest generates Exception on success

### ISSUE THREAD

**Issue by Micsou** [eversign/eversign-php-sdk#11]
Hi, 
I'm trying to send a reminder via the API. 
It looks that it works well but ApiRequest->startRequest() can't parse success json repsonse... 
It's dum. 
here's the dump of the Json repsonse and Exception thrown. 

![2018-06-03_13h57_41](https://user-images.githubusercontent.com/4601910/40886340-334187ea-6736-11e8-9d39-78088488d69d.png)

Thanks. 
Michael

**Comment by dominikkukacka** [comment]
Could you provide me with the code you used to trigger this error?
Which version of the SDK are you using?
Which PHP version are you using?

This excerpt from `example/document-operations.php` works fine:
```
use Eversign\Client;

$config = require('./config.php');

$client = new Client($config['accessKey'], $config['businessId']);
$document = $client->getDocumentByHash($config['documentHash']);
$signers = $document->getSigners();
foreach ($signers as $signer) {
    $client->sendReminderForDocument($document, $signer);
}
```

**Comment by dominikkukacka** [comment]
nvm, I just could reproduce it. will look into it ASAP.

**Comment by Micsou** [comment]
Hi Dominik

PHP 7.0.29 

My Code is like that... 

```
    protected function getEversignClient()
    {
        $societeBusinessId =  $this->getSocieteBusinessId();
        if ($societeBusinessId == null){
            throw new \Exception("Le service Eversign n'est pas configuré pour la société !");
        }

        $client = new Client($this->apiKey, $societeBusinessId);
        return $client;
    }

    public function sendReminder(Document $document, Signer $signer)
    {
        $client = $this->getEversignClient();
        $client->sendReminderForDocument($document, $signer);
    }
```

**Comment by dominikkukacka** [comment]
We just released 1.10.0 which should fix this: https://packagist.org/packages/eversign/eversign-php-sdk

**Comment by Micsou** [comment]
Great, many thanks for your responsiveness !
